### PR TITLE
Game input and state updates

### PR DESCRIPTION
**Pull request by juan-insuasti**
Addresses #10

### CODE REVIEW

**Review by josemariogutierrez — APPROVED** [review]

File: ProjectAwesome/Assets/Scripts/GameState.cs [hunk 1]
```diff
@@ -8,18 +8,50 @@ public class GameState : ScriptableObject
...
+
+    public void ResetHits() {
+        hits = 0;
+    }
+
+    public int GetHits() {
+        return hits;
     }
-}
+}
```
Comment (camigomezdev) [inline ProjectAwesome/Assets/Scripts/GameState.cs:61]: please remove spaces

Comment (juan-insuasti) [reply ProjectAwesome/Assets/Scripts/GameState.cs:61]: `Really?

File: ProjectAwesome/Assets/Scripts/InputController.cs [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+    string prevArrowKey;
+
+    private void Start()
+    {
+        state.ResetHits();
+    }
+
+    void Update()
+    {
+        if (state.gameStateName == "PLAY")
```
Comment (camigomezdev) [inline ProjectAwesome/Assets/Scripts/InputController.cs]: Please, save "PLAY"  word in a constant

File: ProjectAwesome/Assets/Scripts/InputController.cs [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+        else {
+            prevArrowKey = "";
+            return false;
+        }
+    }
+
+    bool CheckArrow(string arrowValue) {
+        string currentArrow;
+        float axisX = Input.GetAxis(axis + "X");
+        float axisY = Input.GetAxis(axis + "Y");
```
Comment (camigomezdev) [inline ProjectAwesome/Assets/Scripts/InputController.cs]: Same here

File: ProjectAwesome/Assets/Scripts/InputController.cs [hunk 4]
```diff
@@ -0,0 +1,164 @@
...
+
+        prevArrowKey = "";
+        return false;
+    }
+
+    bool CheckTool(int ToolCode) {
+        float button0 = Input.GetAxis(axis + "A");
+        float button1 = Input.GetAxis(axis + "B");
+        float button2 = Input.GetAxis(axis + "C");
+        float button3 = Input.GetAxis(axis + "D");
```
Comment (camigomezdev) [inline ProjectAwesome/Assets/Scripts/InputController.cs]: And here

**Review by camigomezdev — APPROVED** [review]

**Review by jusk02 — APPROVED** [review]

**Pull request merged**